### PR TITLE
[fbrp] no escape

### PR DESCRIPTION
**Pull request by lshamis**
# Description

Add `fbrp.NoEscape` to disable shell quoting for sub commands.

TODO: maybe rename it `LateBinding`, or something like that....
TODO: get it working with env keys

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [ ] Refactor
- [x] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [x] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

## Before and After

See `fbrp/examples/07_rosmsg/fsetup.py`

# Testing

`fbrp/examples/07_rosmsg/fsetup.py` + `fbrp/examples/09_cpp_ompl/fsetup.py` + unit test

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [x] I have added tests that show that the PR is functional.
- [x] New and existing unit tests pass locally with my changes.
- [x] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I have verified that all scripts in `tests/scripts` runs on hardware.

### CODE REVIEW

File: fbrp/tests/fbrp/runtime/test_conda.py [hunk 1]
```diff
@@ -8,84 +7,146 @@
...
+            proc_def=proc_def,
+            args=mock_namespace,
+        )
         os_env_patch = mock.patch.dict(os.environ, {"my_path": "path"})
         os_env_patch.start()
         conda_env = await launcher.activate_conda_env()
         mock_file.assert_called_with(f"/tmp/fbrp_conda_test_conda.env")
-        self.assertTrue(len(conda_env) == 1)
-        self.assertDictEqual(conda_env, {"env_var" : "data"})
+        assert len(conda_env) == 1
```
Comment (girifb) [inline fbrp/tests/fbrp/runtime/test_conda.py:81]: curious, why the change from `self.assertTrue` to `assert` ?

**Review by girifb — APPROVED** [review]

File: fbrp/tests/fbrp/runtime/test_conda.py (hunk 1 above)
Comment (lshamis) [reply fbrp/tests/fbrp/runtime/test_conda.py:81]: I "believe" `self.assertTrue` is part of the built-in unittest library. The pytest module, that we're using to run the tests uses and recommends `assert`: https://docs.pytest.org/en/6.2.x/
